cmd/bucket-lifecycle-handlers.go

```diff
@@ -33,6 +33,8 @@ import (
 const (
 	// Lifecycle configuration file.
 	bucketLifecycleConfig = "lifecycle.xml"
+	// Expiry Lifecycle configuration file
+	expiryBucketLifecycleConfig = "expiry-lifecycle.xml"
```

Comment:
may want to add a field in Lifecycle config to track last update timestamp for expiry rules. Otherwise, there will be redundant lifecycle configs and consistency problems.

Reply:
Ok. Will update the PR accordingly.